File: 217-ContainsDuplicate/217-ContainsDuplicate.cpp
// Last updated: 6/9/2025, 1:56:38 PM
class Solution {
public:
    bool containsDuplicate(vector<int>& nums) {
        unordered_set<int> s;
        for(int i=0;i<nums.size();i++)
        {
            if(s.find(nums[i])!=s.end())
            {
                return true;

            }
            s.insert(nums[i]);
        }
        return false;

    }
};

File: 217-ContainsDuplicate/217-ContainsDuplicate.js
// Last updated: 6/9/2025, 1:56:32 PM
/**
 * @param {number[]} nums
 * @return {boolean}
 */
var containsDuplicate = function(nums) {
        const s = new Set();
        for (let i = 0; i < nums.length; i++) {
            if (s.has(nums[i])) {
                return true;
            }
            s.add(nums[i]);
        }
        return false;    
};